src/components/Pill/index.jsx

```diff
@@ -40,7 +40,7 @@ const Pill = ({ statuses }) => {
     }
   });
   return pills.map(pill => (
-    <span className={classNames('ml-2', pill.className)}>
+    <span className={classNames('ml-2', pill.className)} key={pill.text}>
```

Comment:
If I'm understanding this correctly, I don't think this will ensure uniqueness unfortunately since you can have multiple course runs in the same state.

Reply:
They just need to be unique in the iterator, and the text shouldn't be the same for multiple pills on the same collapsible right?

Reply:
Ah okay, I misunderstood this uniqueness constraint. Yes, each collapsible should have unique pills.